Assets/LeapMotion/Core/Scripts/Splines/SplineUtil.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Leap.Unity.Infix;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Splines {
+
+  using IPositionSpline = ISpline<Vector3, Vector3>;
+  using IPoseSpline = ISpline<Pose, Movement>;
+  public static class SplineUtil {
+
+    #region Conversion Utils
+
+    public static PoseToPositionSplineWrapper AsPositionSpline(this IPoseSpline poseSpline) {
```

Comment:
Is there any pose  spline that doesn't already implement ISpline<Vector3, Vector3> ?  This doesn't seem to be used anywhere.

Reply:
>pose spline that doesn't already implement ISpline<Vector3, Vector3>
Pose Splines are ISpline<Pose, Movement>, this converts them to being used for their position only, ignoring rotation info. This is one of those things I used for something else and then wound up not using that something else. Could remove, but I do think it's useful

Reply:
Roger, makes sense